app/components/agenda/agenda-header/agenda-actions.js

```diff
@@ -352,6 +353,43 @@ export default class AgendaAgendaHeaderAgendaActions extends Component {
     this.router.refresh(this.router.currentRouteName);
   }
 
+  get canEmptyInternalReviews() {
+    // action will do nothing on designAgenda A, so hide it instead
+    const isDesignAgendaA = this.args.currentAgenda.status.get('isDesignAgenda') && this.args.currentAgenda.serialnumber === 'A';
+    return this.currentSession.may('manage-agendaitems') && !isDesignAgendaA;
+  }
+
+  emptyInteralReviews = async() => {
+    this.showConfirmEmptyInternalReviews = false;
+    this.args.onStartLoading(this.intl.t('empty-internal-review'));
+    // getting all valid agendaitems first gets better results than trying submission-internal-review directly via subcase
+    const approvedAgendaitems = await this.store.queryAll('agendaitem', {
+      'filter[:has:previous-version]': true,
```

Comment:
This won't work with a single, approved agenda

Reply:
Tom fixed this during initial pair review